swagger/v1/openapi.json

```diff
@@ -1,12 +1,12 @@
 {
   "swagger": "2.0",
   "info": {
-    "title": "Insights Compliance API V1",
+    "title": "Cloud Services for RHEL Compliance API V1",
     "version": "v1",
-    "description": "This is the API for Insights Compliance. You can find out more about Red Hat Insights at [https://access.redhat.com/products/red-hat-insights/](https://access.redhat.com/products/red-hat-insights/)"
+    "description": "This is the API for Cloud Services for RHEL Compliance. You can find out more about Red Hat Cloud Services for RHEL at [https://cloud.redhat.com/](https://cloud.redhat.com/)"
   },
   "paths": {
-    "/r/insights/platform/compliance/profiles": {
+    "/api/compliance/profiles": {
```

Comment:
Any way to use env variables here?

Reply:
This file is auto-generated from the specs, the ENV variables are taken into account directly when running `rake rswag:specs:swaggerize`